src/clients/Builder.ts

```diff
@@ -0,0 +1,21 @@
+import { Builder as BuilderBase } from '@vtex/api'
+
+const SECOND = 1000
+const builderHubTimeout = 2 * SECOND
+
+const builderBaseRoute = `/_v/builder/0`
+const routes = {
+  tsConfig: `${builderBaseRoute}/tsconfig`,
+  typings: `${builderBaseRoute}/typings`,
+}
+
+export default class Builder extends BuilderBase {
```

Comment:
You don't need creating a new client. Update the node-vtex-api project with the new routes and methods to use them.